src/Sulu/Bundle/RouteBundle/Resources/config/doctrine/BaseRoute.orm.xml

```diff
@@ -19,6 +19,11 @@
                 <option name="default">1970-01-01 00:00:00</option>
             </options>
         </field>
+        <field name="changed" type="datetime" nullable="false">
```

Comment:
Well, maybe it is a BC break, but before that it would not be able to upgrade for the same reason it is not working in the Route example. When thinking of it, these lines mean that it will automatically set the `changed` and `created` date if they are not applied, isn't that exactly what the `NOT NULL` should prevent? Is there maybe a solution to execute a few lines of SQL to fill some value only for the existing routes, leaving the functionality intact for routes added later?